blog/keycloak-newsletter-1.ftl

```diff
@@ -1,53 +1,53 @@
 <#--
 title = Keycloak Community Newsletter #1
-date = 2019-03-11
-publish = false
+date = 2019-03-26
+publish = true
 author = Sébastien Blanc
 -->
 
 <p>
-intro
+This is the very first "Keycloak Community Newsletter". The goal of this newsletter is to share the news around the Keycloak project : contributions, technical articles, future plans of the team etc...  
 </p>
 
 <h2>News from the community</h2>
 <p>
-<ul>
-    <li>item one</li>
-    <li>item two</li>
-</ul>
+Since the beginning of the year, the community has been really active. Each week several blog posts speaking about Keycloak are published, here is a short selection.
 </p>
-
-<h2>News from the project</h2>
 <p>
-<ul>
-    <li>item one</li>
-    <li>item two</li>
-</ul>
+Let's start with Philip Riecks who explains in <a href="https://rieckpil.de/howto-microprofile-jwt-authentication-with-keycloak-and-react/">this article</a> how you can leverage Microprofile JWT Authentication with Keycloak and React.
 </p>
-
-<h2>News from the Identity Management World</h2>
 <p>
-<ul>
-    <li>item one</li>
-    <li>item two</li>
-</ul>
+Ramandeep Singh has been <a href="https://medium.com/@ramandeep.singh.1983/enterprise-web-app-authentication-using-keycloak-and-node-js-c10b0e26b80d">blogging</a> about Keycloak and NodeJS and Joshua Alfred Erney explain in this <a href="https://www.jerney.io/secure-apis-kong-keycloak-1/">blog serie</a> how to integrate Keylcoak and <a href="https://konghq.com/">Kong</a>, a popular API management platform.
```

Comment:
You are referring to two different things in same sentence. Split into separate paragraphs or at least separate sentences.